src/data/markdown/docs/03 cloud/04 Project and Team Management/04 Azure AD SAML SSO.md

```diff
@@ -0,0 +1,108 @@
+<br><br>
+
+6. Give the application a name, eg. **k6 Cloud**.
+<br><br>
+
+7. Click **"Add"**.
+<br><br>
+
+8. When your application has successfully been added, click **"Set up single sign on"** (or the equivalent "Single sign-on" link in the left menu):
+
+    ![Azure AD Setup Single Sign-On](images/04 Azure AD SAML SSO/azure-ad-setup-single-sign-on.png)
+<br><br>
+
+9. Click **"SAML"** to enable it:
+
+    ![Azure AD Enable SAML](images/04 Azure AD SAML SSO/azure-ad-enable-saml.png)
+<br><br>
+
+10. Edit **"Basic SAML Configuration"**:
+
+    ![Azure AD Basic SAML Configuration](images/04 Azure AD SAML SSO/azure-ad-setup-basic-config.png)
+
+    Setting:
+
+    | Property                                   | Value                               |
+    | ------------------------------------------ | ----------------------------------- |
+    | Identifier (Entity ID)                     | `https://api.k6.io/sso/acs/`        |
+    | Reply URL (Assertion Consumer Service URL) | `https://api.k6.io/sso/acs/`        |
+    | Logout Url                                 | `https://app.k6.io/account/logout` |
+
+    Resulting in:
+
+    ![Azure AD Basic SAML Configuration](images/04 Azure AD SAML SSO/azure-ad-setup-basic-config2.png)
+<br><br>
+
+11. Edit **"User Attributes & Claims"**:
+
+    ![Azure AD User Attributes](images/04 Azure AD SAML SSO/azure-ad-setup-user-attributes.png)
+
+    Setting the following user attributes (and clearing the "Namespace" property for each attribute):
+
+    | Attribute                | Value                                                                     |
+    | -------------------------| ------------------------------------------------------------------------- |
+    | `Unique User Identifier` | `user.userprincipalname`                                                  |
+    | `user.email`             | `user.userprincipalname`                                                  |
+    | `user.username`          | `user.userprincipalname`                                                  |
+    | `user.first_name`        | `user.givenname`                                                          |
+    | `user.last_name`         | `user.surname`                                                            |
+    | `token`                  | An unique token that you'll be provided with by the k6 Cloud support team. |
+
+    Resulting in:
+
+    ![Azure AD User Attributes](images/04 Azure AD SAML SSO/azure-ad-setup-user-attributes2.png)
+<br><br>
+
+12. Copy the "App Federation Metadata Url" and send it to k6 Cloud support for completion of the setup.
+
+    ![Azure AD SAML Signing Certificate](images/04 Azure AD SAML SSO/azure-ad-setup-saml-signing-cert.png)
+<br><br>
+
+13. Once you've gotten confirmation from k6 Cloud support that your account is ready we advise you to test the integration by clicking the "Test" button in Azure AD:
```

Comment:
An step before this, is to assign users/groups to the `k6 Cloud` app they created on Azure. Otherwise the integration won't work. (returns error)

Reply:
I'll add that as a step before 13). I was taking the position of assuming that anyone wanting SAML SSO already knows how it works in their IdP, but it's better to be extra clear than assume things.